### ISSUE TITLE
HTTP is not a class

### ISSUE THREAD

**Issue by scottbarrow** [stefansundin/github-release-party#1]
Just trying out the gem but on `rake deploy` i receive 
```rake aborted!
TypeError: HTTP is not a class
```

**Comment by stefansundin** [comment]
Hi. Thanks for trying out the gem. Can you please run `rake deploy --trace` to get a stack trace?

Also, please make sure you are running the latest version (0.3.3).

**Comment by scottbarrow** [comment]
Actually I'm getting this on starting rails server now
```/scott/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/github-release-party-0.3.3/lib/http.rb:7:in `<main>': HTTP is not a class (TypeError)```


```rake deploy --verbose
rake aborted!
TypeError: HTTP is not a class
/config/application.rb:20:in `<top (required)>'
/Rakefile:2:in `require_relative'
/Rakefile:2:in `<top (required)>'
(See full trace by running task with --trace)
```

I think your HTTP class is clashing with net/http. I succeeded by renaming your class

**Comment by stefansundin** [comment]
I see now that the README had instructions that caused the old version to be installed. I think if you change the line in your `Gemfile` to use `require: false` then this problem will go away. Then make sure you require the rake task in your `Rakefile`.

**Comment by scottbarrow** [comment]
I tried 0.3.3 with `require: false` and the issue remains. As mentioned above if I rename your `HTTP` class this issue is resolved

**Comment by stefansundin** [comment]
It is strange since the `net/http` class is `Net::HTTP` so it shouldn't clash.  Anyway, thanks for reporting, hopefully I can fix it.

**Comment by stefansundin** [comment]
This should be fixed now with v0.3.4. Removed most of that garbage code. :)
